CHANGES.txt

```diff
@@ -3,6 +3,7 @@ Merged from 3.0:
  * Ensure legacy rows have primary key livenessinfo when they contain illegal cells (CASSANDRA-15365)
 Merged from 2.2:
  * In-JVM DTest: Set correct internode message version for upgrade test (CASSANDRA-15371)
+ * Update nodetool help stop output (CASSANDRA-?????)
```

Comment:
are you sure about those question marks ????

Reply:
Yes, they are there until I raise the request and I know the number.